src/components/organisms/details-collapsible/index.tsx

```diff
@@ -0,0 +1,26 @@
+import * as RadixCollapsible from "@radix-ui/react-collapsible"
```

Comment:
Should we streamline our radix-ui imports? i.e. always use something like `Radix[componentName]` for clarity?

Reply:
I like this idea! Don't think we should prioritize it for the already merged features, but let's see if we can make it happen for future PRs 👍